<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>711</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Day</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Hour</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End Hour</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="changeHoursBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Change hours</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
